values.yaml

```diff
@@ -467,6 +467,10 @@ concourse:
       ##
       sharedPath:
 
+      ## Path templates to use for credential lookup.
+      ##
+      lookupTemplates: ["/{{.Team}}/{{.Pipeline}}/{{.Secret}}", "/{{.Team}}/{{.Secret}}"]
```

Comment:
I'm iffy on having a default here but it looks like that's what we do elsewhere. 🤔

Reply:
What happens if it's blank? does the binary have a default embedded in it?

Reply:
Yep. Should I just remove this?

Reply:
(Yes.)